tests/Microsoft.Bot.Builder.Tests/Teams/TeamsActivityHandlerTests.cs

```diff
@@ -1512,6 +1513,108 @@ public async Task TestMessageDeleteActivityTeams_NoChannelData()
+
+            List<Activity> activitiesToSend = new List<Activity>();
+            void CaptureSend(Activity[] arg)
+            {
+                activitiesToSend.AddRange(arg.ToList());
+            }
+
+            var turnContext = new TurnContext(new SimpleAdapter(CaptureSend), activity);
+
+            // Act
+            var bot = new TestActivityHandler();
+            await ((IBot)bot).OnTurnAsync(turnContext);
+
+            // Assert
+            Assert.Equal(2, bot.Record.Count);
+            Assert.Equal("OnInvokeActivityAsync", bot.Record[0]);
+            Assert.Equal("OnTeamsMessageSubmitActionAsync", bot.Record[1]);
+            Assert.NotNull(activitiesToSend);
+            Assert.Equal(2, activitiesToSend.Count);
+            Assert.Equal(JsonConvert.SerializeObject(activity.Value), activitiesToSend[0].Text);
+            Assert.IsType<InvokeResponse>(activitiesToSend[1].Value);
+            Assert.Equal(200, ((InvokeResponse)activitiesToSend[1].Value).Status);
+        }
+
+        [Fact]
+        public async Task TestMessageFetchTaskDefault()
+        {
+            // Arrange
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "message/fetchTask"
+            };
+
+            Activity[] activitiesToSend = null;
+            void CaptureSend(Activity[] arg)
+            {
+                activitiesToSend = arg;
+            }
+
+            var turnContext = new TurnContext(new SimpleAdapter(CaptureSend), activity);
+
+            // Act
+            var bot = new TestActivityHandler();
+            await ((IBot)bot).OnTurnAsync(turnContext);
+
+            // Assert
+            Assert.Single(bot.Record);
+            Assert.Equal("OnTeamsMessageFetchTaskAsync", bot.Record[0]);
+            Assert.NotNull(activitiesToSend);
+            Assert.Single(activitiesToSend);
+            Assert.IsType<InvokeResponse>(activitiesToSend[0].Value);
+            Assert.Equal(200, ((InvokeResponse)activitiesToSend[0].Value).Status);
+        }
+
+        [Fact]
+        public async Task TestMessageFetchTaskCustom()
+        {
+            // Arrange
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "message/fetchTask"
+            };
+
+            Activity[] activitiesToSend = null;
+            void CaptureSend(Activity[] arg)
+            {
+                activitiesToSend = arg;
+            }
+
+            var turnContext = new TurnContext(new SimpleAdapter(CaptureSend), activity);
+
+            // Act
+            var bot = new TestActivityHandler();
+            await ((IBot)bot).OnTurnAsync(turnContext);
+
+            // Assert
+            Assert.NotNull(activitiesToSend);
+            Assert.Single(activitiesToSend);
+            Assert.IsType<InvokeResponse>(activitiesToSend[0].Value);
+            Assert.Equal(200, ((InvokeResponse)activitiesToSend[0].Value).Status);
```

Comment:
Let's add these assertions to differentiate this test from the previous one.
```suggestion
            // Assert
            Assert.Equal(2, bot.Record.Count());
            Assert.Equal("OnInvokeActivityAsync", bot.Record[0]);
            Assert.Equal("OnTeamsMessageFetchTaskAsync", bot.Record[1]);
            Assert.NotNull(activitiesToSend);
            Assert.Single(activitiesToSend);
            Assert.IsType<InvokeResponse>(activitiesToSend[0].Value);
            Assert.Equal(200, ((InvokeResponse)activitiesToSend[0].Value).Status);
```

Reply:
Changed the two suggestions, thanks